src/frontend/Sider/Journalføring/Standard/Journalføring.tsx

```diff
@@ -78,60 +88,121 @@ interface Props {
-                            journalpostState={journalpostState}
-                        />
-                    </section>
-                    <section>
-                        <Heading spacing size={'small'} level={'2'}>
-                            Behandling
-                        </Heading>
-                        <Behandlinger
-                            journalpostState={journalpostState}
-                            settFeilmelding={settFeilmelding}
-                        />
-                    </section>
-                    {feilmelding && <Alert variant="error">{feilmelding}</Alert>}
-                </InnerContainer>
-            </Venstrekolonne>
-            <Høyrekolonne>
-                <PdfVisning journalpostState={journalpostState} />
-            </Høyrekolonne>
-        </Kolonner>
+        <>
+            <Kolonner>
+                <Venstrekolonne>
+                    <InnerContainer>
+                        <section>
+                            <Heading spacing size={'medium'} level={'1'}>
+                                Journalføring
+                            </Heading>
+                            <JournalpostPanel
+                                journalpost={journalResponse.journalpost}
+                                journalpostState={journalpostState}
+                            />
+                        </section>
+                        <section>
+                            <Heading spacing size={'small'} level={'2'}>
+                                Dokumenter
+                            </Heading>
+                            <Dokumenter journalpostState={journalpostState} />
+                        </section>
+                        <section>
+                            <Heading spacing size={'small'} level={'2'}>
+                                Bruker
+                            </Heading>
+                            <BrukerPanel journalpostResponse={journalResponse} />
+                        </section>
+                        <section>
+                            <Heading spacing size={'small'} level={'2'}>
+                                Avsender
+                            </Heading>
+                            <AvsenderPanel
+                                journalpostResponse={journalResponse}
+                                journalpostState={journalpostState}
+                            />
+                        </section>
+                        <section>
+                            <Heading spacing size={'small'} level={'2'}>
+                                Behandling
+                            </Heading>
+                            <Behandlinger
+                                journalpostState={journalpostState}
+                                settFeilmelding={settFeilmelding}
+                            />
+                        </section>
+                        {feilmelding && <Alert variant="error">{feilmelding}</Alert>}
```

Comment:
Bruk gjerne `Feilmelding` og at feilmelding er undefined i stedet for en tom streng
```suggestion
                        <Feilmelding variant="error">{feilmelding}</Feilmelding>}
```